config/locales/en.yml

```diff
@@ -645,16 +645,21 @@ en:
       email_body: |
         Hello %{preferred_name},
 
-        Thanks for starting your taxes with GetYourRefund - you’re almost there! We will prepare your taxes once we have all of your required information and tax documents.
+        Thanks for starting your taxes with GetYourRefund - you’re almost there! Your case number is %{client_id}. We will prepare your taxes once we have all of your required information and tax documents.
```

Comment:
We call these confirmation numbers elsewhere. Are you sure we're supposed to call them case numbers here? If so, that's fine, but thought I'd ask. Non-blocking.